documentation/devonfw-ide-introduction.asciidoc

```diff
@@ -1,12 +1,73 @@
-= Introduction
+= devonfw-ide
 
-http://devonfw.com[devonfw] provides a solution to building applications which combine best-in-class frameworks and libraries
-as well as industry proven practices and code conventions.
+== What is devonfw-ide?
+
+`devondw-ide` is a standardized, open-source toolbox or construction kit allowing you to pick and choose those tools which fit the requirements and needs of your project.
+
+It is not another Eclipse. It gives you Eclipse, along with Java, maven, NodeJs, VSCode and everything else you may need to quick start your development. All you need to do is download and extract a zip file and execute its setup, reducing your initial setup time for starting a project from days to minutes.
+
+That is not all. `devonfw-ide` gives you its own command shell where you can execute `devon` commands. Here is a small video which showcases what you can do with `devonfw-ide`:
```

Comment:
I would see it extends the existing shell with helpful commands to automate the provisioning of the development environment.

Reply:
Modified accordingly.